oada-core/libs/oada-lib-kafka/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// prettier-ignore
+import type { EventEmitter } from 'events'
+
+declare class Base extends EventEmitter {
+}
+
+interface KafkaReguest {
```

Comment:
```suggestion
interface KafkaRequest {
```

Reply:
I named it that one purpose. They are guests.